testsuite/forge/src/backend/local/swarm.rs

```diff
@@ -106,7 +107,7 @@ impl LocalSwarm {
     where
         R: ::rand::RngCore + ::rand::CryptoRng,
     {
-        println!("Building a new swarm");
+        info!("Building a new swarm");
```

Comment:
Do we know how this affects the `forge-cli` in local mode? i.e., is a logger already created so that these are visible? They're required for manual verification/testing, see: https://github.com/aptos-labs/aptos-core/tree/audit/testsuite/forge-cli/src

Reply:
we do ::aptos_logger::Logger::new().init(); in the first line of build() function, so it should be there